<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugInterface</class>
 <widget class="QMainWindow" name="DebugInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debugging Tools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/pics/resources/pics/Beacon.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="ColorLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="statusTip">
           <string>#FE8F01</string>
          </property>
          <property name="whatsThis">
           <string>#FE8F01</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(254, 143, 1);</string>
          </property>
          <property name="text">
           <string>â—¼</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="statusTip">
           <string>#FE8F01</string>
          </property>
          <property name="whatsThis">
           <string>#FE8F01</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(254, 143, 1);</string>
          </property>
          <property name="text">
           <string>Amber</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="ProcessInfo">
       <attribute name="title">
        <string>Process Statement</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="LibDisplay"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="BuildInfo">
       <attribute name="title">
        <string>Build Info</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QTextBrowser" name="BuildInfoDisplay"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Fonts">
       <attribute name="title">
        <string>Fonts</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="FontDisplay"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="AboutContent">
       <attribute name="title">
        <string>About Content</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QTextBrowser" name="AboutContentDisplay"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="GetAbout">
          <property name="text">
           <string>Get</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/pics/resources/pics/Author.png</pixmap>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
